doc/langref/test_pointer_arithmetic.zig

```diff
@@ -11,6 +11,9 @@ test "pointer arithmetic with many-item pointer" {
     // slicing a many-item pointer without an end is equivalent to
     // pointer arithmetic: `ptr[start..] == ptr + start`
     try expect(ptr[1..] == ptr + 1);
+
+    // subtraction between any two pointers except slices is supported
+    try expect(ptr[1] - ptr[0] == 1);
```

Comment:
This is just subtracting two `i32`'s and works in status-quo Zig - did you mean to do something like `try expect(ptr[1..] - ptr == @sizeOf(i32));`

Reply:
Yeah that's what I meant to do